ibm/data_source_ibm_is_flow_log.go

```diff
@@ -209,7 +209,9 @@ func dataSourceIBMIsFlowLogRead(context context.Context, d *schema.ResourceData,
 		start := ""
 		allrecs := []vpcv1.FlowLogCollector{}
 		for {
```

Comment:
This for loop can be removed and pagination logic can be removed when name is given as we can directly hit the list api get with name